.github/workflows/build-unified.yml

```diff
@@ -0,0 +1,40 @@
+name: build-unified
+on:
+  push:
+    paths-ignore: # don't rebuild if only documentation has changed
+      - "**.md"
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  build-individual:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.generate-matrix.outputs.matrix }}
```

Comment:
I'm having a hard time understanding where this value is coming from

Reply:
same

Reply:
Oh, nowhere. Good catch. That's a leftover from when I tried to hook into the GitHub Actions matrixing to build all the separate images for the modules. (it got scrapped because it was hard to develop but didn't offer much speed benefits) (https://github.com/blue-build/modules/pull/240/commits/de067312273bd55b8b919dbcd90161e48e9a464e)

Reply:
Fixed in [a1b2017](https://github.com/blue-build/modules/pull/240/commits/a1b201731e809188ca6129fdd0ed46048d779f88)